11.Portret.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<title>Document</title>
+</head>
+<body>
+	<script>
+		const year = prompt("Напишіть свій рік народження");
+		const city = prompt("В якому місті ви живете: Київ, Вашингтон, Лондон або в іншому?");
+		const sports = prompt("Ваш улюлений вид спорту футбол, баскетбол або бокс?");
+
+		const Result = () => {
+			const cityResult = () => {
+				if (city === "Київ") {
+					return ('Ви живете в столиці України')
+				}
+		        if (city === "Вашингтон") {
+		            return ('Ви живете в столиці Сполучених Штатів Америки')
+		        }
+		        if (city === "Лондон") {
+					return ('Ви живете в столиці Англії')
+		        } 
+		            return (`Ви живете в місті ${city}`)
+			}
+			cityResult()
+
+			const sportResult = () => {
+				if(sports === "футбол") {
+		            return ('Круто! Хочеш стати як Кріштіану Роналду?')
+			    }
+			    if(sports === "баскетбол") {
+			        return ('Круто! Хочеш стати як Майкл Джордан?')
+			    }
+			    if(sports === "бокс") {
+			        return ('Круто! Хочеш стати як Майк Тайсон?')
+			    } 
+			}
+			sportResult()
+
+			const emptyString = () => {
+				if (year === (null) || (year === (''))) {
```

Comment:
Move this condition to `yearResult`

You don't have `yearResult`, so create it